contracts/package.json

```diff
@@ -37,10 +37,10 @@
     "@ethersproject/bignumber": "~5.7.0",
     "@ethersproject/contracts": "~5.7.0",
     "@ethersproject/providers": "~5.7.2",
+    "@nomicfoundation/hardhat-chai-matchers": "^1.0.6",
```

Comment:
Hey @RensR was this supposed to be `^2.0.6` (latest)?
With `1.0.6` I'm running into an issue when trying to use hardhat:
```
Error HH801: Plugin @nomicfoundation/hardhat-chai-matchers requires the following dependencies to be installed: @nomiclabs/hardhat-ethers.
```
Which conflicts with the current `"@nomicfoundation/hardhat-ethers": "^3.0.5"`